curriculum/challenges/english/03-front-end-development-libraries/redux/use-const-for-action-types.md

```diff
@@ -87,12 +87,10 @@ The `authReducer` function should handle multiple action types with a switch sta
 
 ```js
 const noWhiteSpace = __helpers.removeWhiteSpace(code);
-assert(
-  (/constLOGIN=(['"`])LOGIN\1/.test(noWhiteSpace) &&
-    /constLOGOUT=(['"`])LOGOUT\1/.test(noWhiteSpace)) ||
-      /const(LOGIN|LOGOUT)=(['"`])\1\2,(?!\1)(LOGIN|LOGOUT)=(['"`])\3\4/.test(noWhiteSpace) ||
-        /const\[(LOGIN|LOGOUT),(?!\1)(LOGIN|LOGOUT)\]=\[(['"`])\1\3,\3\2\3\]/.test(noWhiteSpace)
-);
+assert(LOGIN === 'LOGIN' && LOGOUT === 'LOGOUT')
+assert(noWhiteSpace.includes('const'))
+assert.isNotOk(noWhiteSpace.includes('let'))
+assert.isNotOk(noWhiteSpace.includes('var'))
```

Comment:
You would need to use the helper to strip comments I think

![image](https://user-images.githubusercontent.com/26656284/180425162-efb155aa-cde6-40ae-b39b-2f6b240eac52.png)

If instead it's not important that users would fail adding comments, LGTM